Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -30,9 +30,9 @@ bool BatteryStateOfChargeService::isCharging() const
 QTime BatteryStateOfChargeService::timeWhenChargedOrDepleted() const
 {
     if(isCharging_){
-        return holdC;
+        return timeUntilCharged;
     }else{
```

Comment:
Try to keep open and closed curly brackets on separate lines for readability.
like so: 
`return timeUntilCharged;`
`}`
`else { `